dist/htmlminifier.js

```diff
@@ -848,7 +857,9 @@
       return attrValue;
     }
     else if (isMetaViewport(tag, attrs) && attrName === 'content') {
-      attrValue = attrValue.replace(/1\.0/g, '1').replace(/\s+/g, '');
+      attrValue = attrValue.replace(/\s+/g, '').replace(/[0-9]+\.[0-9]+/g, function(num) {
+        return (+num).toString();
```

Comment:
I would add a comment just to make it clear what's going on "0.9000" -> "0.9", "1.0" -> "1", "1.0001" -> "1.0001"